README.md

```diff
@@ -1,2 +1,23 @@
 # promote-demo
 A sample microservice to be used with the `services' gitops-cli
+# Example source repository to be promoted using GitOps
+
+Note the directory structure - this is a *source repository* and not a GitOps repository. It's not intended to be a production-grade application by any means - the point is that it's something to build and promote, and nothing more.
+
+This can be promoted into an existing remote GitOps repository with, for example (do this from the parent directory):
+
+```
+services promote --from promote-demo --to https://github.com/<myorg>/<mydevopsrepository>.git --service promote-demo
+```
+
+an examplar *GitOps* repository can be found at https://github.com/rhd-gitops-example/gitops-example-staging.
+
+So what happens here, with the promote?
+
+https://docs.google.com/document/d/102LiAQBc0Mb3ZNSguL1ZhCojvEXOGGtzUdub-e58Mog/edit# outlines the design in more detail, but specifically (for the local "source to GitOps" repository), this happens:
```

Comment:
Our README in the [Overview](https://github.com/rhd-gitops-example/services/blob/master/README.md#services--) links to the now-external version of this doc, [here](https://github.com/rhd-gitops-example/docs/tree/master/model#gitops-repository). Please can you update to link to one of these two?

Reply:
Done